package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node index.js"
+    "start": "nodemon index.js"
```

Comment:
Could you make this `node run dev` instead? So when we actually do `npm start` on deployment, it doesn't use `nodemon`.

Reply:
Done